src/components/Typewriter.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Typewriter(props) {
```

Comment:
if it's Component please use CamelCase `TypeWriter`

Reply:
Done